react-native-libraries.json

```diff
@@ -9685,5 +9685,16 @@
     "ios": true,
     "android": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/jakex7/react-native-click-outside",
+    "examples": [
+      "https://github.com/jakex7/react-native-click-outside#-usage",
```

Comment:
Thanks for another addition, LGTM! 👌

Let's remove Readme link from examples array before the merge:
```suggestion
```

Reply:
Thanks 🔥